### ISSUE TITLE
Remove legacy PACKAGE.py files

### ISSUE THREAD

**Issue by egede** [ganga-devs/ganga#2221]
In the different plugins of Ganga there is a `PACKAGE.py` that back in ancient history defined dependencies. I believe the files themselves and the loading of them and subsequent creation of of instances of the `PackageSetup` class can all be deleted.
```
./GangaCore/PACKAGE.py
./GangaDirac/PACKAGE.py
./GangaGaudi/PACKAGE.py
./GangaLHCb/PACKAGE.py
./GangaND280/PACKAGE.py
./GangaTest/Lib/GListApp/PACKAGE.py
./GangaTest/Lib/TFile/PACKAGE.py
./GangaTest/PACKAGE.py
./GangaTutorial/PACKAGE.py
```

**Comment by souravpy** [comment]
I would love to pick up the issue, can I know where are `PackageSetup` class defined, so that I can delete them along with afore mentioned `PACKAGE.py` files. Thank you !!

**Comment by egede** [comment]
Just check in the import statements where it is located. Or do a search across the source code. 

Thanks for considering fixing this.

**Comment by souravpy** [comment]
Should I delete` PackageSetup `class across the source code or should we avoid subsequent creations of the class?

**Comment by souravpy** [comment]
PR deleting all PACKAGE.py files #2247

**Comment by mesmith75** [comment]
Fixed by #2247
